bin/build.sh

```diff
@@ -365,6 +366,15 @@ if [ ${INSTALL_OBJECTIVEC} -eq 1 ] ; then
   puts "Installed Objective-C"
 fi 
 
+if [ ${INSTALL_RUST} -eq 1 ] ; then
+  ${WGET} https://static.rust-lang.org/dist/${RUST_VERSION}/rust-nightly-x86_64-unknown-linux-gnu.tar.gz -O rust.tar.gz
```

Comment:
Add `puts Installing Rust` to match other installation blocks. Then this should be done. Squash your commits and I'll merge them.

Reply:
Done